Userland/Libraries/LibCore/LocalSocket.cpp

```diff
@@ -55,25 +58,33 @@ LocalSocket::~LocalSocket()
 
 pid_t LocalSocket::peer_pid() const
 {
-#ifdef AK_OS_MACOS
+#if defined(__APPLE__)
```

Comment:
Like @ADKaster commented, it would be nice to have some sort of AK_OS macro here.